automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -197,13 +198,16 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
     //https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/nbTranslate
     "should install user specified notebook extensions" in {
```

Comment:
that's one notebook *extension*, not plural, right?

Reply:
True, this test is just installing 1 extension. I can rename it.